### ISSUE TITLE
Additional Whitespace

### ISSUE THREAD

**Issue by jneilliii** [jneilliii/OctoPrint-ConsolidatedTabs#2]
When panels are moved the page retains full height and therefore there is a lot of additional whitespace left on the page.

**Comment by jneilliii** [comment]
This may be fixed in that latest 0.0.8 branch, needs more testing.

**Comment by jneilliii** [comment]
Solved with current gridstack development branch.

```
https://github.com/jneilliii/OctoPrint-ConsolidatedTabs/archive/develop/gridstack.zip
```
